packages/typescript/core/tests/clients/artifacts/databricks.test.ts

```diff
@@ -365,40 +371,100 @@ describe('DatabricksArtifactsClient', () => {
       });
     });
 
-    const azureTypes = ['AZURE_SAS_URI', 'AZURE_ADLS_GEN2_SAS_URI'];
+    it('should successfully upload to Azure Blob Storage with AZURE_SAS_URI', async () => {
+      const traceInfo = new TraceInfo({
+        traceId: 'tr-AZURE_SAS_URI',
+        traceLocation: {
+          type: TraceLocationType.MLFLOW_EXPERIMENT,
+          mlflowExperiment: { experimentId: '0' }
+        },
+        state: TraceState.OK,
+        requestTime: 12000
+      });
 
-    azureTypes.forEach((type) => {
-      it(`should throw not implemented error for ${type}`, async () => {
-        const traceInfo = new TraceInfo({
-          traceId: `tr-${type}`,
-          traceLocation: {
-            type: TraceLocationType.MLFLOW_EXPERIMENT,
-            mlflowExperiment: { experimentId: '0' }
-          },
-          state: TraceState.OK,
-          requestTime: 12000
-        });
+      const traceData = new TraceData([]);
 
-        const traceData = new TraceData([]);
+      server.use(
+        http.get(
+          `${testHost}/api/2.0/mlflow/traces/tr-AZURE_SAS_URI/credentials-for-data-upload`,
+          () => {
+            return HttpResponse.json({
+              credential_info: {
+                type: 'AZURE_SAS_URI' as any,
+                signed_uri: 'https://storage.azure.com/file?sas=token'
+              }
+            });
+          }
+        )
+      );
 
-        server.use(
-          http.get(
-            `${testHost}/api/2.0/mlflow/traces/tr-${type}/credentials-for-data-upload`,
-            () => {
-              return HttpResponse.json({
-                credential_info: {
-                  type: type as any,
-                  signed_uri: 'https://storage.azure.com/file?sas=token'
-                }
-              });
-            }
-          )
-        );
+      // Mock Azure Blob Storage upload
+      server.use(
+        http.put('https://storage.azure.com/file', () => {
+          return new HttpResponse(null, { status: 201 });
+        })
+      );
 
-        await expect(client.uploadTraceData(traceInfo, traceData)).rejects.toThrow(
-          `Azure upload not yet implemented for credential type: ${type}`
-        );
+      await client.uploadTraceData(traceInfo, traceData);
+
+      // Test passes if no error is thrown
+      expect(true).toBe(true);
```

Comment:
Let's avoid `expect(true).toBe(true);`
```suggestion
      await expect(async () => {
          await client.uploadTraceData(traceInfo, traceData);
      }).not.toThrow();
```